lib/decimal.mli

```diff
@@ -324,6 +324,9 @@ val to_bool : t -> bool
 val to_rational : t -> Q.t
 val to_string : ?eng:bool -> ?context:Context.t -> t -> string
 
+val to_decimal_string : t -> string
+(** [to_decimal_string t] is the decimal representation of [t], i.e. assuming [t] is finite, the only characters in the string are ['0'..'9'] and ['.']. *)
```

Comment:
Ahhh isn't this slightly not true for `NaN` and `Infinity`?

Reply:
That's why I said "assuming `t` is finite"